packages/blocks/openops-tables/src/actions/get-table-url-action.ts

```diff
@@ -0,0 +1,27 @@
+import { BlockAuth, createAction } from '@openops/blocks-framework';
+import {
+  getTableIdByTableName,
+  openopsTablesDropdownProperty,
+} from '@openops/common';
+import { SharedSystemProp, system } from '@openops/server-shared';
+
+const DEFAULT_DATABASE_ID = 1;
+
+export const getTableUrlAction = createAction({
+  auth: BlockAuth.None(),
+  name: 'get_table_url',
+  description: 'Get a shareable URL for the provided table.',
+  displayName: 'Get Table URL',
+  props: {
+    tableName: openopsTablesDropdownProperty(),
+  },
+  async run(context) {
+    const tableName = context.propsValue.tableName as unknown as string;
+    const tableId = await getTableIdByTableName(tableName);
+    const baseUrl = system.getOrThrow(SharedSystemProp.FRONTEND_URL);
+
+    return (
+      baseUrl + `/tables?path=/database/${DEFAULT_DATABASE_ID}/table/${tableId}`
```

Comment:
same logic as we do
https://github.com/openops-cloud/openops/blob/ad603dad8ae2d77d8d65f79577814105088bb735/packages/react-ui/src/app/routes/home-demo/index.tsx#L44

This will allow sending a link on slack that leads to our full app, and the table opened
<img width="1142" alt="image" src="https://github.com/user-attachments/assets/9b33c0ad-d90b-4d29-ad2c-82cc7e42b7ad" />